rest-server/src/config/auth.js

```diff
@@ -47,5 +47,5 @@ const authSchema = Joi.object().keys({
 module.exports = {
   secret: config.jwtSecret,
   check: jwtCheck,
-  schema: authSchema
+  schema: authConfigSchema
```

Comment:
this schema is used for parameter validation instead of server config

Reply:
ok. Will change.